app/models/manageiq/providers/openshift/monitoring_manager.rb

```diff
@@ -1,22 +1,22 @@
-module ManageIQ::Providers
-  class Openshift::MonitoringManager < ManageIQ::Providers::Kubernetes::MonitoringManager
-    require_nested :EventCatcher
+ManageIQ::Providers::Kubernetes::MonitoringManager.include(ActsAsStiLeafClass)
```

Comment:
This one is weird, because it seems the base monitoring manager didn't have ActsAsStiLeafClass, but maybe it should have?

Reply:
Yeah I think this one was missed